data/org.freedesktop.portal.Documents.xml

```diff
@@ -273,5 +273,21 @@
       <arg type='s' name='app_id' direction='in'/>
       <arg type='a{say}' name='docs' direction='out'/>
     </method>
+
+    <!--
+        GetRealPath:
+        @doc_id: the ID of the file in the document store
+        @path: the path for the file in the host filesystem
+
+        Gets the filesystem path for a document store entry.
+
+        This call is available inside the sandbox, if the application has the permission for the document.
+
+        This method was added in version 5 of this interface.
+    -->
+    <method name="GetRealPath">
```

Comment:
I'll leave this to others to decide, but I think better naming would be just `HostPath` instead `GetRealPath`.